dom/src/DOMSourceable.ts

```diff
@@ -0,0 +1,7 @@
+import {Sourceable} from './Sourceable';
+export {GenericStream} from './GenericStream';
+export {EventsFnOptions} from './EventsFnOptions';
+
+export interface DOMSourceable extends Sourceable {
```

Comment:
As per chat and diagrams on Gitter, this should now have been fixed. Thank you for taking your time to review this.